api/python/quilt3/workflows/config-1.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://quiltdata.com/workflows/config/1",
+  "type": "object",
+  "required": [
+    "version",
+    "workflows"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "version": {
+      "const": "1"
+    },
+    "is_workflow_required": {
+      "type": "boolean",
+      "description": "Whether package can be pushed with unspecified workflow.",
```

Comment:
That's not quite correct: if user doesn't pass workflow and there is default, this default will be used.
It could be something like: "If true, any pushed package must conform to some workflow."

Reply:
I'm not sure that's the right logic. If `is_workflow_required=true` that means ANY one workflow is required. Automatically falling back on `default` is to magical and may confuse people. If a user fails one workflow and pushes again, that's a separate event against a potentially separate workflow.

Reply:
I was saying that if there is default workflow set, user doesn't have to "select and pass a workflow" (do you mean pass a workflow to API?), because that default workflow will be used (implicitly selected) for validation.

Reply:
Oh, by "pass a workflow" I mean "succeed a workflow". Edited.

Reply:
But still, users don't have to select a workflow if there is the default one, have they?

Reply:
Rewrote